### PR TITLE
Retitle Web APIs, part 1

### PR DESCRIPTION
**Pull request by wbamberg**
Part 1 of https://github.com/mdn/mdn/issues/284.

This retitles the first 50 Web/API interfaces.

### CODE REVIEW

File: files/en-us/web/api/audiodecoder/audiodecoder/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: AudioDecoder.AudioDecoder()
+title: "AudioDecoder.AudioDecoder: AudioDecoder.AudioDecoder() constructor"
```
Comment (wbamberg) [inline files/en-us/web/api/audiodecoder/audiodecoder/index.md:2]:
```suggestion
title: "AudioDecoder: AudioDecoder.AudioDecoder() constructor"
```

File: files/en-us/web/api/audiodecoder/audiodecoder/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+short-title: AudioDecoder.AudioDecoder()
```
Comment (wbamberg) [inline files/en-us/web/api/audiodecoder/audiodecoder/index.md:3]:
```suggestion
short-title: AudioDecoder()
```

File: files/en-us/web/api/audioencoder/audioencoder/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-title: AudioEncoder.AudioEncoder()
+title: "AudioEncoder.AudioEncoder: AudioEncoder.AudioEncoder() constructor"
+short-title: AudioEncoder.AudioEncoder()
```
Comment (wbamberg) [inline files/en-us/web/api/audioencoder/audioencoder/index.md:3]:
```suggestion
title: "AudioEncoder: AudioEncoder() constructor"
short-title: AudioEncoder()
```

File: files/en-us/web/api/audioworkletprocessor/parameterdescriptors/index.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-title: AudioWorkletProcessor.parameterDescriptors (static getter)
+title: "AudioWorkletProcessor: parameterDescriptors (static getter) property"
+short-title: parameterDescriptors (static getter)
```
Comment (wbamberg) [inline files/en-us/web/api/audioworkletprocessor/parameterdescriptors/index.md:3]:
```suggestion
title: "AudioWorkletProcessor: parameterDescriptors property"
short-title: parameterDescriptors
```

**Review by teoli2003 — APPROVED** [review]
👍 Approving but not merging as it is the first of its kind, and others may want to review them.

**Review by Elchi3 — APPROVED** [review]
Looks good and `short-title` works well! 👍

**Comment by hamishwillee** [comment]
FWIW I agree ^^^^

**Comment by wbamberg** [comment]
Yeah, I like these, and it will be great to get this work done, finally!

**Pull request merged**